config/install/field.field.paragraph.item_list.field_il_view_mode.yml

```diff
@@ -19,6 +19,7 @@ default_value_callback: ''
 settings:
   exclude:
     stack_simple: stack_simple
+    calendar_item: calendar_item
```

Comment:
@abhisekmazumdar , should this be added in? I think this is being removed from this ticket. https://redmine.morpht.com/issues/36801

cc: @tanashin-kishimoto

Reply:
PR was outdated.